### DISCUSSION TITLE
Examples

### DISCUSSION THREAD

**Discussion by RAN-Vestjylland** [BlazorFluentUI/BlazorFluentUI#274] [Ideas]
Hi all,

I'm not sure whether this page is made by this group of developers: https://blazorfluentui.azurewebsites.net/
If it is, i really miss more examples to understand better how the various controls are working really well.

My biggest issues to get working are the following controls:

- BFUCallout
- BFUDialog
- BFUModal

Hope that either the above mentioned page can be updated or sub .md pages can be made to show examples of all the really awesome looking controls.

All the best from Denmark :-)

**Comment by vnbaaij** [comment]
Yes, that page belongs to this group too, but it is running a bit older version of the library. 

The latest released version (4.0.3) can be found at https://www.blazorfluentui.net/. For vNext (5.0) I've added the 'Show code' option to all sample pages and made it all look a bit cleaner. This can be found at the following (temporary) location: https://white-dune-0ef5c1e03.azurestaticapps.net/ These two sites both run on the Blazor clientside implementation, but for seeing and copying the example code that should not matter. Oh, just a heads up, for the v5 version of the library, we've dropped the BFU prefix.

Let us know if you have troubles getting the samples to work in your own environment!

**Reply by vnbaaij** [reply]
PSA: https://www.blazorfluentui.net is now running on V5

**Reply by limefrogyank** [reply]
Oh, that home page is quite beautiful. 👍

**Reply by RAN-Vestjylland** [reply]
Hi, is it possible to get a little more code example for CommandBar with buttons and NavBar with buttons? Thank you in advance.
